docs/workflow_tasks/lzards_backup.md

```diff
@@ -0,0 +1,43 @@
+---
+id: lzards_backup
+title: LZARDS Backup
+hide_title: false
+---
+
+The LZARDS backup task takes an array of granules and initiates backup requests to the LZARDS API, which will be handled asynchronously by LZARDS.
+
+## Deployment
+
+The LZARDS backup task is not automatically deployed with Cumulus. Rather, in order to deploy the task through the Cumulus module, you must specify a `lzards_launchpad_passphrase` in your terraform variables. Additionally, ensure that you are passing that variable by ensuring that the Cumulus module has the following input value configuration:
+
+```
+lzards_launchpad_passphrase  = var.<launchpad_passphrase_variable_name>
+```
```

Comment:
I would suggest these changes to make it a bit more clear:

The LZARDS backup task is not automatically deployed with Cumulus. To deploy the task through the Cumulus module, first you must specify a `lzards_launchpad_passphrase` in your terraform variables (e.g. `variables.tf`) like so:

```
variable "lzards_launchpad_passphrase" {
  type    = string
  default = ""
}
```

Then you can specify a value for your `lzards_launchpad_passphrase` in `terraform.tfvars` like so:

```
lzards_launchpad_passphrase = your-passphrase
```

Lastly, you need to make sure that the `lzards_launchpad_passphrase` is passed into the Cumulus module (in `main.tf`) like so:

```
lzards_launchpad_passphrase  = var.lzards_launchpad_passphrase
```